src/components/binding/binding-panel.js

```diff
@@ -213,13 +187,44 @@ genotet.BindingPanel.prototype.updateTracks = function() {
       this.container.find('#genes #track-' + index).remove();
     }
   }
+
+  this.signal('loadBindingList');
+
+  this.container.find('#genes .glyphicon-remove')
+    .css('display', this.data.tracks.length == 1 ? 'none' : '');
+};
+
+/**
+ * Updates the track list for panel.
+ */
+genotet.BindingPanel.prototype.updateTrackList = function() {
```

Comment:
The key point is not about having one function or two. The key point is how you have the entire process coded cleanly across multiple source files and functions. 

I thought they can be combined because I saw DOM removal in the updateTracks(), while I assume updateTrackList() is the DOM handler. Had the code be better written, it would self-explain the entry point of each atomic process. When the code is unavoidably somewhat misleading, comments need to be added to explain the relation between function calls, i.e. what are the entry points, how functions are chained, etc.